### PR TITLE
Tekken tag 2 aspect ratio patch 

### PR DESCRIPTION
**Pull request by Flextape05**
tekken tag 2 21:9 32:9 48:9 patch

### CODE REVIEW

File: Resolutions/TekkenTagTournament2_Resolution/patches.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[tekken32]
+moduleMatches = 0x465EA719
+0x10433450 = .float $width/$height
+0x101798a4 = .float (0.017453292 * (($width/$height)/($gameWidth/$gameHeight)))
+0x1072d978 = .float (0.017453292 * (($width/$height)/($gameWidth/$gameHeight)))
```
Comment (Crementif) [inline Resolutions/TekkenTagTournament2_Resolution/patches.txt:5]: Uh, could you maybe add some code comments on what each line does?

Comment (Flextape05) [reply Resolutions/TekkenTagTournament2_Resolution/patches.txt:5]: Ok sure

Comment (Flextape05) [reply Resolutions/TekkenTagTournament2_Resolution/patches.txt:5]: ehm i dont know how to change the files i already uploaded but the first one is culling and the second one is props culling

Comment (Flextape05) [reply Resolutions/TekkenTagTournament2_Resolution/patches.txt]: i fix it

**Pull request merged**